assets/js/print.js

```diff
@@ -18,13 +18,22 @@
 
   $.printPreview = {
 
-    printPreview: function() {
+    printPreview: function(element) {
       var $body = $('body'),
           $container = $('.container-main'),
           footnoteLinks = "",
           linksIndex = 0;
 
       $body.find('.nav-mobile, .drilldown, .nav-main, .header-separator, .nav-service, .nav-lang, .form-search, .yamm--select, header > div:first-child, footer, .alert, .icon--print, .social-sharing, form, .nav-process, .carousel-indicators, .carousel-control, .breadcrumb, .pagination-container').remove();
+
+      // if an element is passed, we want it to be the only thing to print out
+      if (element) {
+        element = $('[data-print=' + element + ']');
```

Comment:
redefining an argument is a bad practice. Your linter should say No ;)
By the way, I'm sure to understood the nature of the `element` function argument. Why is it not directly a DOMElement object (or a jQuery object) ?